src/game_display.hpp

```diff
@@ -95,9 +95,15 @@ class game_display : public display
 
 	/**
 	 * Add more paths to highlight.  Print numbers where they overlap.
-	 * Used only by Show Enemy Moves.
+	 * Used by Show Enemy Moves.  If goal is not null_location, highlight
```

Comment:
Add some Doxygen commands.

```suggestion
	 * Used by Show Enemy Moves.  If @a goal is not @c null_location, highlight
```

Reply:
Changed.